test/Core.Test/AutoFixture/Attributes/InlineCustomAutoDataAttribute.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Xunit;
+using Xunit.Sdk;
+using AutoFixture.Xunit2;
+
+namespace Bit.Core.Test.AutoFixture.Attributes
+{
+    internal class InlineCustomAutoDataAttribute : CompositeDataAttribute
+    {
+        public InlineCustomAutoDataAttribute(Type[] iCustomizationTypes, params object[] values)
+        : base(new DataAttribute[] {
```

Comment:
Same, or if too long, indent. We also use 2 x 2 formatting for extended constructor parameters and/or types.